### ISSUE TITLE
Prettify HTML

### ISSUE THREAD

**Issue by bacloud14** [bacloud14/online_arules#2]
all in title!
Probably add a Python rendering engine. Make it sexy but sober ! Is this possible ? :)

**Comment by JChrisWolfe** [comment]
Hi, so this git repo requires knowledge of Flask framework, ML with Python, and the [Efficient-Apriori](https://pypi.org/project/efficient-apriori/)?

**Comment by bacloud14** [comment]
yes @JChrisWolfe but very basic knowledge ... Do you operate in Flask a little ? I am not a web "MVC" guy anyway; if you can do rendering with a Flask engine, you are more than welcome !

**Comment by bacloud14** [comment]
Any follow up? Do you do styling @JChrisWolfe ?
I'm doing some cleaning of my repos..
Many thanks.

**Comment by JChrisWolfe** [comment]
At this time I decided to follow other projects I I'm not too familiar with
machine learning yet and it will take me some time to learn Flask since I'm
used to Django.

Although technically yes I could you could easily do it with a combination
of normal CSS and using a CSS framework like bootstrap or materialize. You
should consider trying it, it saves a lot of time

**Comment by ydocsgnillats** [comment]
I'd like to help with this issue if it is available @bacloud14

**Comment by bacloud14** [comment]
@ydocsgnillats Sure! it is yours, do it your way as I have no style preference.
Thank you for reviving this issue, I totally forgot about this repository and probably will work on it more these days.
Actually I was 100% on the other client side apriori implementation in this repo: https://github.com/bacloud14/js_arules 
Thanks !

**Comment by bacloud14** [comment]
if you don't mind, I read readme again and see also a little thing: 

- Pretty print results (rules)

- Customize parameters on client side (should let the user experience min_support and min_confidence at a reasonable amounts

if you could consider these..

thanks again,

**Comment by ydocsgnillats** [comment]
@bacloud14  That sounds great! I have some questions just to make sure I'm on the right track. I see that this is using flask as the framework, is the html currently hosted somewhere, or just locally for development for now? Also, do you need any other pages added besides index.html, or do you just need that to be redesigned/updated? 

As for the other two things, you just want the results of the upload printed in a nice formatted way on the client side? And you want min_support and min_confidence to be able to be changed from the client side? 

Thank you!

**Comment by bacloud14** [comment]
Hey 👋
It was hosted with a free service provider (Oracle) probably 1 semester ago. Then it went down. I will find some host for this after I work on security part as this is very prone to errors (file size, file type...).
Yep, it is using flask. I understand the client web protocols quite well but not necessarily Python, but at that time, I needed a server and just picked Flask.

Thanks 👍👍

**Comment by JChrisWolfe** [comment]
You may wish to ask someone else as lately I have not been in the code
repository for a long while. I would not be able to find time to do the
project even if I agree.

Godspeed.

**Comment by bacloud14** [comment]
Hi @JChrisWolfe  
You let me know from day 0 that you will not work on this, feel free man! This is GitHub

**Comment by ydocsgnillats** [comment]
I think I have it done, about to send a pull request, let me know what you think!

**Comment by bacloud14** [comment]
@ydocsgnillats thanks this is awesome, However I think static files are not being served by Flask server. I am trying to correct it somehow I am looking here https://flask.palletsprojects.com/en/1.1.x/quickstart/#static-files

Anyway, if you already know-how, or I can try myself,.

thanks again

**Comment by bacloud14** [comment]
ok, I did it, it is being server by web server.

Please, feel free to add a footer referencing us both.

**Comment by bacloud14** [comment]
@ydocsgnillats 
Do you know why I cannot add any new section? I am sure this is because of CSS probably how you styled the html or body tag.
For example, I am trying to add a footer. One beautiful like one of these: https://freefrontend.com/css-headers-footers/

thanks !

**Comment by ydocsgnillats** [comment]
Hmm, it may be something to do with the way I set up the background image to fit the whole page. I can try to add it if you want me to! Is there a specific one you were wanting?

**Comment by bacloud14** [comment]
Sure, feel free to add anything with your taste. Again referencing your Github and mine if you want to.

Thanks

**Comment by bacloud14** [comment]
Something simple like:

```
<div class="footer">
	  <ul>
		<li><a href="https://github.com/bacloud14/js_arules">See on GitHub</a></li>
		<li><a href="https://datasciencetoday.net/index.php/en-us/machine-learning/179-association-rules">Read about Association rules</a></li>
		<li><a href="https://github.com/seratch/apriori.js">apriori.js </a></li>
                <li><a href="https://github.com/ydocsgnillats">Styled by ydocsgnillats</a></li>
		<li>
		  <p>👋</p>
		</li>
	  </ul>
	</div>
```


```
.footer {
  display: flex;
  background-color: #002a38;
  line-height: 1.3;
  font-family: Menlo, monospace;
}

@keyframes wave-animation {
  0%,
  100% {
    transform: rotate(0deg);
  }
  25% {
    transform: rotate(20deg);
  }
  75% {
    transform: rotate(-15deg);
  }
}
```

* actually, I tried this footer in another project.
